.github/workflows/self-hosted-linux-arm64-graviton2-tests.yml.md

```diff
@@ -0,0 +1,136 @@
+
+### Step 2. Install github action on the host
+
+SSH into the instance, and install the github action self-hosted runner (see [install scripts](https://github.com/etcd-io/etcd/settings/actions/runners/new?arch=arm64&os=linux)).
+
+### Step 3. Configure github action on the host
+
+SSH into the instance, and configure the github action self-hosted runner.
+
+First, we need disable ICU install (see [actions/runner issue on ARM64](https://github.com/actions/runner/issues/629)):
+
+```
+sudo yum install -y patch
+```
+
+And write this bash script:
+
+```
+#!/bin/bash -e
+
+patch -p1 <<ICU_PATCH
+diff -Naur a/bin/Runner.Listener.runtimeconfig.json b/bin/Runner.Listener.runtimeconfig.json
+--- a/bin/Runner.Listener.runtimeconfig.json	2020-07-01 02:21:09.000000000 +0000
++++ b/bin/Runner.Listener.runtimeconfig.json	2020-07-28 00:02:38.748868613 +0000
+@@ -8,7 +8,8 @@
+       }
+     ],
+     "configProperties": {
+-      "System.Runtime.TieredCompilation.QuickJit": true
++      "System.Runtime.TieredCompilation.QuickJit": true,
++      "System.Globalization.Invariant": true
+     }
+   }
+-}
+\ No newline at end of file
++}
+diff -Naur a/bin/Runner.PluginHost.runtimeconfig.json b/bin/Runner.PluginHost.runtimeconfig.json
+--- a/bin/Runner.PluginHost.runtimeconfig.json	2020-07-01 02:21:22.000000000 +0000
++++ b/bin/Runner.PluginHost.runtimeconfig.json	2020-07-28 00:02:59.358680003 +0000
+@@ -8,7 +8,8 @@
+       }
+     ],
+     "configProperties": {
+-      "System.Runtime.TieredCompilation.QuickJit": true
++      "System.Runtime.TieredCompilation.QuickJit": true,
++      "System.Globalization.Invariant": true
+     }
+   }
+-}
+\ No newline at end of file
++}
+diff -Naur a/bin/Runner.Worker.runtimeconfig.json b/bin/Runner.Worker.runtimeconfig.json
+--- a/bin/Runner.Worker.runtimeconfig.json	2020-07-01 02:21:16.000000000 +0000
++++ b/bin/Runner.Worker.runtimeconfig.json	2020-07-28 00:02:19.159028531 +0000
+@@ -8,7 +8,8 @@
+       }
+     ],
+     "configProperties": {
+-      "System.Runtime.TieredCompilation.QuickJit": true
++      "System.Runtime.TieredCompilation.QuickJit": true,
++      "System.Globalization.Invariant": true
+     }
+   }
+-}
+\ No newline at end of file
++}
+ICU_PATCH
+```
+
+And now patch the github action runner:
+
+```
+cd ${HOME}/actions-runner
+bash patch.sh
+```
+
+```
+patching file bin/Runner.Listener.runtimeconfig.json
+patching file bin/Runner.PluginHost.runtimeconfig.json
+patching file bin/Runner.Worker.runtimeconfig.json
+```
+
+And now configure:
+
+```
+sudo yum install -y wget
+INSTANCE_ID=$(wget -q -O - http://169.254.169.254/latest/meta-data/instance-id)
+echo ${INSTANCE_ID}
+
+# get token from https://github.com/etcd-io/etcd/settings/actions/runners/new?arch=arm64&os=linux
+cd ${HOME}/actions-runner
+./config.sh \
+--work "_work" \
+--name ${INSTANCE_ID} \
+--labels self-hosted,linux,ARM64,graviton2 \
+--url https://github.com/etcd-io/etcd \
+--token ...
+```
+
+And run:
+
+```
+# run this as systemd service
```

Comment:
could you please provide an examplar systemd `*.conf` file ?

Reply:
Leave it as TODO. We will follow up with the example config file.

Reply:
Actually, no need to add systemd service file. Github action has built-in script to do that:

```
sudo ./svc.sh install
sudo ./svc.sh start
sudo ./svc.sh status
```

Updated the README.